.github/workflows/ci.yml

```diff
@@ -423,24 +425,35 @@ jobs:
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-6-non-generic
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-7-non-generic
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-8-non-generic
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-tpcds
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-oauth2
+            jdk: 11
```

Comment:
reoder entries here, so they match ordering in `matrix`, so that
ordering of labels (eg at https://github.com/trinodb/trino/runs/3795199838  ) is consistent

ie you may just want to move `jdk` in `matrix` as last.

Reply:
yes, I've spotted that but wanted for CI to complete first